src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -212,7 +226,14 @@ def create_privacy_request(
                     },
                 )
                 queue_privacy_request(privacy_request.id)
-
+        except EmailDispatchException as exc:
```

Comment:
I think it's possible for this exception to be thrown when `FIDESOPS__EXECUTION__IDENTITY_VERIFICATION_REQUIRED` is toggled off? if so we should check whether to enforce it in this or otherwise move this `try / except` block closer to `_send_verification_code_to_user`?

Reply:
I don't think we're triggering an email send anywhere else in the flow right now?

Reply:
We'll do it via tasks too as part of #1088, so maybe it's best to leave this for then